.github/workflows/build-test-image.yaml

```
@@ -613,7 +667,9 @@ jobs:
 
       - name: Install dependencies
         if: matrix.instances.if == 'true' || matrix.instances.if == true
-        run: poetry install
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2088:warning:1:14: Tilde does not expand in quotes. Use $HOME

(shellcheck)